app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type annotations for all function parameters and the return type. You have correctly annotated the `friends` parameter and the return type, but the `cafe` parameter is missing its type annotation. Please add it. This is a code style issue.